src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/DiagnosticDescriptorCreationAnalyzer.cs

```diff
@@ -27,6 +27,18 @@ public sealed class DiagnosticDescriptorCreationAnalyzer : DiagnosticAnalyzer
         private const string DiagnosticCategoryAndIdRangeFile = "DiagnosticCategoryAndIdRanges.txt";
         private static readonly (string? prefix, int start, int end) s_defaultAllowedIdsInfo = (null, -1, -1);
 
+        private static readonly ImmutableHashSet<string> CADiagnosticIdAllowedAssemblies = ImmutableHashSet.Create(
+            StringComparer.Ordinal,
+            "Microsoft.CodeQuality.Analyzers",
```

Comment:
Actually there is one more to be added to this list: https://github.com/dotnet/roslyn-analyzers/blob/master/nuget/Microsoft.CodeAnalysis.VersionCheckAnalyzer/Microsoft.CodeAnalysis.VersionCheckAnalyzer.Package.csproj#L16

https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.VersionCheckAnalyzer/VersionCheckAnalyzer.cs#L16

Reply:
Just for additional sanity, can you locally build a NuGet package with this analyzer (execute `eng\common\cibuild.cmd` to generate `Microsoft.CodeAnalysis.Analyzers.nupkg` in artifacts directory) and verify that the new rule does not fire on a project with reference to FxCop analyzers package?

Reply:
That was a good idea! We missed `Text.Analyzers`. I have also added the C# and VB.NET projects even if they don't have any ID yet.

Reply:
Actually with https://github.com/dotnet/roslyn-analyzers/pull/3139, we will also have `Microsoft.CodeAnalysis.NetAnalyzers` and its C# and VB versions. Can you add those as well or at least file a tracking issue to add them?

Reply:
Done